Neos.Neos/Classes/Controller/Backend/ContentController.php

```diff
@@ -150,14 +150,15 @@ public function uploadAssetAction(Asset $asset, string $metadata, string $node,
         $contentRepositoryId = SiteDetectionResult::fromRequest($this->request->getHttpRequest())
             ->contentRepositoryId;
         $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
-        $nodeAddress = NodeAddressFactory::create($contentRepository)->createFromUriString($nodeAddressString);
+        // todo legacy uri node address notation used. Should be refactored to use json encoded NodeAddress
```

Comment:
The TODO is related to line 149, right?

Reply:
jip a followup will change the whole neos ui and neos.neos to work on the new format we agreed upon.